### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by hexagonrecursion**
There was a small typo. I fixed it. (see diff)

### CODE REVIEW

**Review by tibetiroka — APPROVED** [review]

**Review by EjoThims — APPROVED** [review]

**Review by Hurleveur — APPROVED** [review]

**Review by lumbar527 — APPROVED** [review]

**Comment by Saugia** [comment]
I don't think this is a typo. The phrase "The Council was decided" is a proper phrase, although it is more uncommon compared to the alternative here, which is also correct. It just comes down to how more "archaic" we, or the character, wants to sound here.

**Comment by Hecter94** [comment]
A little earlier in this same file, they use "has" in the same context:
https://github.com/endless-sky/endless-sky/blob/242fff9c550e5d058e1e5a9189e14be139bfd285/data/human/free%20worlds%201%20start.txt#L1007

I think this is just a typo, and we should probably be consistent in usage if nothing else.

**Comment by warp-core** [comment]
> A little earlier in this same file, they use "has" in the same context:
> https://github.com/endless-sky/endless-sky/blob/242fff9c550e5d058e1e5a9189e14be139bfd285/data/human/free%20worlds%201%20start.txt#L1007
> 
> I think this is just a typo, and we should probably be consistent in usage if nothing else. 

While it may be talking about the same event, the context is actually very different.

**Comment by Zireael07** [comment]
Googling "Council was decided" brings up nothing. I would go with either has, for present, or had, for past.

File: data/human/free worlds 1 start.txt [hunk 1]
```diff
@@ -1266,7 +1266,7 @@ mission "FW Pirates: Attack 1"
...
 			`	"Oh, the Senate. Of course. The Senate this, the Senate that," Tomek says mockingly while throwing his head back. "The Senate is in no position to be commanding this war. They do not have the military experience that I have, or that you have JJ. Do you think the Republic Parliament is busying themselves voting on every little fleet movement, or do you think they've put this in the hands of the Navy and told them to win this war? I made my reasons clear to you all many times why I didn't want to convene the Senate, because they would only hinder and handicap us from winning this war, and that's exactly what they're already doing!"`
-			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council was decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
+			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council has decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Comment (EjoThims) [inline data/human/free worlds 1 start.txt:1269]:
```suggestion
			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```

With the archaic "I am decided" usage, saying "was decided" here could imply they no longer are.

Easy enough to eliminate all ambiguity on it entirely.

**Comment by hexagonrecursion** [comment]
Thank you, @EjoThims. I updated the pull request with your suggestion

**Comment by warp-core** [comment]
I can't find anything indicating that this usage of "decided" is considered "archaic".

https://www.collinsdictionary.com/dictionary/english/decided
(Scroll down to find the American English adjective definition.)

https://www.merriam-webster.com/dictionary/decided
(Form 2 is what is used here.)

Here, JJ is emphasising that the Council made the decision to convene the Senate with a strong resolve, with certainty, etc. Going on to imply that Tomek has no right to assert that the choices of the Senate should be disregarded now since the Council only continues to operate because the Senate allows it.

Using the verb form ("the Council decided") doesn't have the same impact.

**Comment by Saugia** [comment]
I wasn't saying "decided" specifically, moreso "was decided". It's used in some english, but it is weird. Hence why I said it wasn't incorrect, but it's not widely accepted. I'm fine if we change it to something else though.

**Review by lumbar527 — APPROVED** [review]
Still approving

File: data/human/free worlds 1 start.txt [hunk 2]
```diff
@@ -1266,7 +1266,7 @@ mission "FW Pirates: Attack 1"
...
 			`	"Oh, the Senate. Of course. The Senate this, the Senate that," Tomek says mockingly while throwing his head back. "The Senate is in no position to be commanding this war. They do not have the military experience that I have, or that you have JJ. Do you think the Republic Parliament is busying themselves voting on every little fleet movement, or do you think they've put this in the hands of the Navy and told them to win this war? I made my reasons clear to you all many times why I didn't want to convene the Senate, because they would only hinder and handicap us from winning this war, and that's exactly what they're already doing!"`
-			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council was decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
+			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Comment (ziproot) [inline data/human/free worlds 1 start.txt:1269]:
```suggestion
			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council was clear that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Removing the word "decided" and replacing it with a synonym should help prevent the confusion. To me these words have the same meaning, but I don't know about other people. I do think that this is a better idea than trying to choose between keeping it as an adjective, making the text confusing, or making it a verb, changing the meeting.

**Comment by Amazinite** [comment]
As Saug and Warp noted, "was decided" isn't actually incorrect, and was intentional on my part.

**Comment by ziproot** [comment]
What's wrong with "The council was clear...?"

**Comment by EjoThims** [comment]
> Here, JJ is emphasising that the Council made the decision to convene the Senate with a strong resolve, with certainty, etc. Going on to imply that Tomek has no right to assert that the choices of the Senate should be disregarded now since the Council only continues to operate because the Senate allows it.


But the usage of the past tense implies that that resolve may have changed; by omission you are leaving open the question of if the council still -is- decided on that point.

That's also why "has decided" here is also correct (though also awkward), because the state of holding that decision is one which still continues even after the event it triggered has also passed. Changing to verb with "had decided" would open the same implication of wavering resolve in the face of Tomek's response.

"It had decided" implies the potential for changing one's mind. "It has decided" creates a persisting condition from a past event.

This is why I also initially believed it to be a simple letter swap typo. All these possibilities (the original included) create awkward sentences, but only "has decided" does so without itself opening up the possible interpretation that some of the Council (or at the very least just JJ) now regret that decision.

> As Saug and Warp noted, "was decided" isn't actually incorrect, and was intentional on my part. 
> 
> https://ludwig.guru/s/was+decided
> 
> https://ludwig.guru/s/it+was+decided+to

These examples seem to be overwhelmingly about events or outcomes being decided -upon-, not people or groups -making- decisions, so don't really make the case that the usage here is currently common (or even correct, though it still is). To match the framing in them, we'd need something like "convening the Senate was decided by the Council."

Either way, the actual intent is still unclear in the original, however, and the passage can be improved either through removing the ambiguity entirely or reframing the sentence.

**Review by EjoThims — APPROVED** [review]

**Comment by ziproot** [comment]
> But the usage of the past tense implies that that resolve may have changed; by omission you are leaving open the question of if the council still -is- decided on that point.
In that case, how about "The Council has decidedly determined that the Senate would intervene." This still shows that they "were decided" as intended, while also showing that they still are decided.

File: data/human/free worlds 1 start.txt (hunk 2 above)
Comment (ziproot) [reply data/human/free worlds 1 start.txt:1269]:
```suggestion
			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council has decidedly determined that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Alternative suggestion: This changes the meaning of the text, in order to show that the Council does not regret their decision, while still showing that the Council was clear when they made the decision. If the goal is to preserve the initial meaning, my initial suggestion ensures that people will be less tripped up by the (correct) grammar.

**Comment by Zireael07** [comment]
I now understand the rationale behind the original phrasing, but it needs a translator comment at the very least! (I am an English major and I thought it was a typo, too!)

**Comment by hexagonrecursion** [comment]
Ouch! I did not expect something that to me looked like a typo to generate this much discussion

I see numerous suggestions:

1. The Council **was decided** (the original by @Amazinite 9eafe444565ad573370ac445bcd0949243ac76fd)
2. The Council **has decided** (my proposal)
3. The Council **decided** (@EjoThims)
4. The Council **was clear** (@ziproot)
5. The Council **has decidedly determined** (also @ziproot)

I am now waiting either for consensus or for someone to say "I am in charge here, here is what we will do"

> I now understand the rationale behind the original phrasing, but it needs a translator comment at the very least! (I am an English major and I thought it was a typo, too!)

@Zireael07, please share your insight with the ~rest of the class~ rest of us. What was the rationale?

**Comment by Zireael07** [comment]
Look up the comment from @EjoThims that cleared it up for me

**Comment by hexagonrecursion** [comment]
@Zireael07, the original phrasing was "The Council **was** decided". @EjoThims  appears to argue against this option

**Comment by Zireael07** [comment]
Oof, then it's more proof that the current situation is confusing - I misread them as being FOR the option

**Comment by EjoThims** [comment]
@Zireael07 

I think the original is a bit confusing in what it implies the possibility of, even when accounting for it being written as intended (which is indeed grammatically correct).

Essentially, none of these options are -wrong- at current, but they all do mean and further imply subtly different things.

I'm favoring simply removing the ambiguity in that meaning over trying to clarify the original intent (especially without an entire overhaul of these couple of sentences, which is necessary to put forth the original intent in a way that doesn't have this same confusion from multiple different angles still being an issue later).

**Comment by Amazinite** [comment]
I'm fine with just dropping "was".

**Pull request merged**